ONSdigital/eq-prototypes
# MBS 0111 prototype build

### Context of this PR

An interactive prototype of the MBS 0111 survey for testing question designs and issues around collection of financial values.

### How to test

1. Configure the test using `/eq-prototypes/mbs/0111/setup/`
2. Route a) using default period values or route b) by defining new date range
3. Enter a value for the total turnover
4. Route with or without 'significant changes in turnover' and proceed to summary screen.
5. Summary should display:
* Date range
* Total turnover with unit and to two decimal places
* Either 'No' for no significant changes or 'Yes', the reasons provided and a comment.
